en/02_Developer_Guides/08_Performance/99_Other.md

```diff
@@ -0,0 +1,20 @@
+---
```

Comment:
Just a tempory page for me to dump idea. Content here will either be moved somewhere else or deleted